### DISCUSSION TITLE
inputs

### DISCUSSION THREAD

**Discussion by RemcoTerol** [dbuezas/arduino-web-oscilloscope#5] [General]
Hi David,

Great application, thanks a lot!!
One question though: what is input AS ?
In the the demo picture it look's like there is a second channel.

with kind regard's

Remco

**Comment by dbuezas** [comment]
Thanks!
Yes, you need to flash a second board with the contents of arduino2 and connect 8 digital pins together. It's been a while so you'll have to look at the code to find out which ones.

**Comment by RemcoTerol** [comment]
thanks in advance

Remco


Op wo 21 dec. 2022 om 14:53 schreef David Buezas ***@***.***>:
